doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -43,41 +43,237 @@ On your computer, one of the following operating systems:
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+.. _nrf54h20_install_ncs:
+
+Installing the |NCS|
+====================
+
+After you have installed the toolchain, complete the following steps to get the customer sampling version of the |NCS|:
+
+1. In the terminal window opened during the toolchain installation, initialize west with the revision of the |NCS| from the customer sampling by running the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west init -m https://github.com/nrfconnect/sdk-nrf --mr v2.6.99-cs2
+
+#. Enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Installing the Terminal application
+***********************************
 
 On your computer, install `nRF Connect for Desktop`_.
 You must also install a terminal emulator, such as `nRF Connect Serial Terminal`_ (from the nRF Connect for Desktop application) or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).
 
+Installing the nRF54H20 firmware bundle
+***************************************
+
+The nRF54H20 SoC requires the installation of a bundle ( :file:`nrf54h20_soc_binaries_v0.3.0.zip`) containing the precompiled firmware for the Secure Domain and System Controller:
+
+..
+   ### Update bundle link
+
+1. Download the `nRF54H20 firmware bundle`_.
+   The bundle is a ZIP archive that contains the following files:
+
+   * :file:`nordic_top.suit`
+   * :file:`suit_installed_envelopes_secure_merged.hex`
+   * :file:`sysctrl.hex`
+   * :file:`urot.hex`
+
+#. Unzip the bundle in a folder of your choice, then indicate the bundle folder path to nRF Util using the following command::
+
+      nrfutil device x-provision-nrf54h --firmware *path-to_bundle_.zip_file* --serial-number *serialnumber*
+
+..
+   ### ADAC Commands
+   ### Flash the sample first
+
 .. _ug_nrf54h20_gs_sample:
 
 Programming the sample
 **********************
 
+..
+   ## check build commands
+
 The :ref:`multicore_hello_world` sample is a multicore sample running on both the Application core (``cpuapp``) and the Peripheral Processor (PPR, ``cpuppr``).
-It uses the ``nrf54h20dk/nrf54h20/cpuapp`` build target.
+It uses the ``nrf54h20dk_nrf54h20_cpuapp`` build target.
 
 To build and program the sample to the nRF54H20 DK, complete the following steps:
 
```

Comment:
Step 1 below: Connect the nRF54H20 DK to you computer using the IMCU USB port on the DK. ->Connect the nRF54H20 DK to your computer using the IMCU USB port on the DK.